includes/shortcodes/class-sensei-shortcode-user-courses.php

```diff
@@ -64,7 +84,19 @@ class Sensei_Shortcode_User_Courses implements Sensei_Shortcode_Interface {
 	 * @param string $shortcode  the shortcode that was called for this instance
 	 */
 	public function __construct( $attributes, $content, $shortcode ) {
-
+		global $wp_query;
+		$attributes = shortcode_atts( array(), $attributes, $shortcode );
```

Comment:
@pgk I was taking a look at the `[sensei_user_courses]` shortcode functionality. I think this line may be a problem as setting the first parameter to `array()` appears to clobber any of the parameters that are added to the shortcode. For example, "status" would be ignored for `[sensei_user_courses status="active"]`.

Reply:
Good catch!